lte/protos/pipelined.proto

```diff
@@ -346,6 +346,7 @@ message SetGroupPDR {
   SetGroupFAR set_gr_far = 8;
   DeactivateFlowsRequest deactivate_flow_req = 9;
   ActivateFlowsRequest activate_flow_req = 10;
+  QCI session_qfi = 11;
```

Comment:
Why is the QFI typed to QCI? My understanding is that the QFI is some dynamic identifier and the QCI classidfies by a fixed value.

Reply:
The QFI may be dynamically assigned or may be equal to the 5QI. So here, QFI (5G) is used as QCI (4G).